docs/cl/libvirt.md

```diff
@@ -0,0 +1,245 @@
+# Libvirt
+
+In this tutorial, boot and provision a Kubernetes v1.11.0 using a local libvirt instance.
+
+The libvirt archirecture is meant for development and testing, rather than long-term use. It does not support load-balancing between multiple API servers, for example.
+
+We'll download a base image, then use Typhoon to directly provision the virtual machines via Terraform.
+
+Controllers are provisioned to run an `etcd-member` peer and a `kubelet` service. Workers run just a `kubelet` service. A one-time [bootkube](https://github.com/kubernetes-incubator/bootkube) bootstrap schedules the `apiserver`, `scheduler`, `controller-manager`, and `coredns` on controllers and schedules `kube-proxy` and `calico` (or `flannel`) on every node. A generated `kubeconfig` provides `kubectl` access to the cluster.
+
+## Requirements
+
+* At least 2 GiB free ram and 20 GiB free disk
+* NetworkManager in dnsmasq mode
+* libvirt / virsh
+* Terraform v0.11.x and 
+    - [terraform-provider-libvirt ](https://github.com/dmacvicar/terraform-provider-libvirt)
+    - [terraform-provider-ct ](https://github.com/coreos/terraform-provider-ct)
+
+## Libvirt
+
+Libvirt is a suite of tools that manages virtual machines, storage, and networking. It does not run virtual machines directly, rather it relies on a lower-level virtualization engine such as qemu or bhyve.
+
+You will need the `virsh` and `qemu-img` binaries installed on your system:
+
+```
+sudo dnf install libvirt-client qemu-img
+```
+
+## CoreOS Base image
```

Comment:
Typhoon consistently refers to these as "Container Linux Images". It accomodates Flatcar and future forks.